### ISSUE TITLE
Images with an empty name breaks the creation of new elastic agents

### ISSUE THREAD

**Issue by goeranu** [gocd-contrib/openstack-elastic-agent#7]
If there exists an image in OpenStack with an empty name, we get a null pointer exception when there is an attempt to create a new agent.

To create an image triggering the problem, simply do
```
openstack image create --public --file … ''
```
This will produce Java backtraces in the go server log.  The interesting part of the backtrace looks like this.
```
Caused by: java.lang.NullPointerException: null
        at cd.go.contrib.elasticagents.openstack.utils.OpenstackClientWrapper.getImageId(OpenstackClientWrapper.java:36)
```
The code looks like this
```
for (Image tmpImage : client.compute().images().list()){
    if (tmpImage.getName().equals(nameOrId)) {
```
I haven't run it in a debugger, but my assumption is that `getName()` will return a null pointer in this case.  The obvious fix would be to check for this before calling `equals()`.

**Comment by chrislovsund** [comment]
We have been using the fix successfully in our Prod.

**Comment by tomzo** [comment]
This is now released in `0.9.0`
